megamek/src/megamek/client/bot/princess/MultiTargetFireControl.java

```diff
@@ -78,6 +79,10 @@ public FiringPlan getBestFiringPlan(final Entity shooter,
 
             if (currentPlan.getUtility() > bestPlan.getUtility()) {
                 bestPlan = currentPlan;
+
+                // Debug logging; Princess does her own info-level logging
+                LogManager.getLogger().debug(shooter.getDisplayName() + " - Best Firing Plan: " +
```

Comment:
As per the other logging comment - this is in a loop so we want to eliminate it.